tests/WE2E/test_configs/special_builds/config.UFS_FIRE.yaml

```diff
@@ -0,0 +1,60 @@
+metadata:
```

Comment:
I'm not sure you want to carve out the space "special builds" for this. We'd presumably have many different "ufs-weather-model app builds" in the future. Maybe it makes sense to put it under a subdir `test_configs/ufs_fire/`, then we may have several test configs for the UFS Fire build, and others under similarly named `test_configs/mpas_dycore`, or `test_configs/aqm`, etc.

Reply:
Your reasoning makes sense, I'll revert this change.